src/modules.js

```diff
@@ -1,17 +1,39 @@
+import axios from "axios";
+import Bowser from "bowser";
+import firebase from "firebase/app";
+import 'firebase/auth';
```

Comment:
R (Required): As mentioned in previous comment it would be import that only if `infermedicaAnalytics` will be used.

Reply:
k, I will fix it